kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/utils/KafkaUtils.java

```diff
@@ -46,15 +46,15 @@ public class KafkaUtils {
     private static final String KAFKA_VERSION_VAR = "%KAFKA_VERSION%";
 
     /**
-     * Consume message string.
+     * Consume message with java client.
      *
      * @param deployNamespace the deploy namespace
      * @param topicName the topic name
      * @param bootstrap the bootstrap
      * @param timeout the timeout
      * @return the log of the pod
      */
-    public static String consumeMessage(String deployNamespace, String topicName, String bootstrap, Duration timeout) {
+    public static String consumeMessageWithJavaClient(String deployNamespace, String topicName, String bootstrap, Duration timeout) {
```

Comment:
I appreciate that this is invoking a java client in the end, but I find it a bit surprising that `consumeMessageWithJavaClient` invokes a shell script in a pod.

Intellij is also suggesting this method is unused so maybe we should just delete it?

Reply:
Agree. It was created in the very first commit of the system tests